crates/matrix-sdk-ui/tests/integration/room_list.rs

```diff
@@ -268,12 +236,6 @@ async fn test_sync_from_init_to_enjoy() -> Result<(), Error> {
                 },
             },
             "extensions": {
-                "e2ee": {
-                    "enabled": true,
-                },
-                "to_device": {
-                    "enabled": true,
-                },
```

Comment:
That's wrong, we should have those :-).

Reply:
Noooo! ... or maybe, you're right, at least on Android :-)

On iOS, the main app will use both the `RoomList` and `EncryptionSync`, so the `RoomList` on iOS mustn't have these two extensions. Maybe we could make it a parameter in the room list ctor? Or have Android spawn both too, at least for symmetry with iOS? (and so it gets some of other benefits: e2ee not being blocked by processing of lists on the ss proxy, etc.)

Reply:
Hmm, let's keep this as is for now :-). Sorry for the confusion!